src/rpdk/core/generate_stack_templates.py

```diff
@@ -0,0 +1,234 @@
+import json
+import re
+import shutil
+from pathlib import Path
+from typing import Any, Dict
+from uuid import uuid4
+
+import yaml
+
+CONTRACT_TEST_FOLDER = "contract-tests-artifacts"
+CONTRACT_TEST_INPUT_PREFIX = "inputs_*"
+CONTRACT_TEST_DEPENDENCY_FILE_NAME = "dependencies.yml"
+FILE_GENERATION_ENABLED = "file_generation_enabled"
+TYPE_NAME = "typeName"
+CONTRACT_TEST_FILE_NAMES = "contract_test_file_names"
+INPUT1_FILE_NAME = "inputs_1.json"
+FN_SUB = "Fn::Sub"
+FN_IMPORT_VALUE = "Fn::ImportValue"
+UUID = "uuid"
+DYNAMIC_VALUES_MAP = {
+    "region": "${AWS::Region}",
+    "partition": "${AWS::Partition}",
+    "account": "${AWS::AccountId}",
+}
+
+
+class StackTemplateGenerator:
```

Comment:
same comment, I dont think you need a class for that

Reply:
We have the flexibility to approach this in different ways. However, my preference is to have StackTemplateGenerator separately, allowing it to function as an independent component. By doing so, we can logically separate both components, resulting in a cleaner and more organized codebase.